microprofile-health/pom.xml

```diff
@@ -49,6 +49,11 @@
             <artifactId>rest-assured</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>com.google.code.gson</groupId>
+            <artifactId>gson</artifactId>
```

Comment:
Where is this artifact used?

Reply:
Resolving, as new changes have been pushed that no longer contain this.